pegasus/sites.v3/code.org/public/statichoc2015/index.haml

```diff
@@ -0,0 +1,76 @@
+---
+layout: none
+theme: none
+title: Hour of Code 2015
+---
+
+:css
+  h4 {
+    margin-bottom: 5px;
+    padding-top:40px;
+  }
+
+  h2 {
+    padding-top:30px;
+  }
+
+  h3 {
+    margin-bottom: 0px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+- tutorials = Tutorials.new(:tutorials)
+
+%a{href: "/mcstatic"}
```

Comment:
I am not doing banners for now, but am randomly changing the order of the first two tutorials.